src/Components/test/E2ETest/Infrastructure/SeleniumStandaloneServer.cs

```diff
@@ -80,6 +65,7 @@ void LogOutput(object sender, DataReceivedEventArgs e)
             {
                 lock (builder)
                 {
+                    Console.WriteLine(e.Data);
```

Comment:
Do we need this? selenium usually has a lot to say.

Reply:
Conditionally enable it with an environment variable?

Reply:
Probably not. I had to change this class a bit because we had what looked like an infinite loop and we weren't caching startup failures. The `Lazy<T>` fixed it